### DISCUSSION TITLE
running dmftproj in parallel: combine case.almblm_*

### DISCUSSION THREAD

**Discussion by jrhui** [TRIQS/dft_tools#184] [Q&A] [answered]
If I run wien2k in k-points parallel mode, there would have many `case.almblm_*` files in the directory, how to combine them into single `case.almblm` file, since `dmftproj` can only run in single core.

**Comment by jkarp314** [comment, marked as answer]
Hi, 

It's my understanding that this is not implemented and dmftproj doesn't work with parallel wien2k. 

If you only want single shot DFT+DMFT, you could run the DFT calculation in parallel to convergence, then use save_lapw to a new directory, run one more iteration in single mode in that directory and then call dmftproj.
